pdsc.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/xml"
+	"errors"
+	"fmt"
+)
+
+type PdscXML struct {
+	XMLName xml.Name `xml:"package"`
+	Vendor  string   `xml:"vendor"`
+	URL     string   `xml:"url"`
+	Name    string   `xml:"name"`
+
+	ReleasesTag struct {
+		XMLName  xml.Name     `xml:"releases"`
+		Releases []ReleaseTag `xml:"release"`
+	} `xml:"releases"`
+}
+
+type ReleaseTag struct {
+	XMLName xml.Name `xml:"release"`
+	Version string   `xml:"version,attr"`
+	Date    string   `xml:"Date,attr"`
+}
+
+func (p *PdscXML) MatchTag(pdscTag PdscTag) error {
+	diff := ""
```

Comment:
I think this is fine, but I may have created a diff struct here and given it a print/toString method.

Reply:
Note taken! I'll leave as is for now, there's a new step of cmsis-pack project and I may create an external utilities pack for shared tag comparison, I'll apply your suggestion then